src/main/scala/it/pagopa/interop/partyprocess/service/impl/GeoTaxonomyServiceImpl.scala

```diff
@@ -48,8 +49,9 @@ final case class GeoTaxonomyServiceImpl(invoker: GeoTaxonomyInvoker, api: Geogra
 
   override def getExtByCodes(
     geoTaxonomycodes: Seq[String]
-  )(implicit contexts: Seq[(String, String)]): Future[Seq[GeographicTaxonomyExt]] =
+  )(implicit contexts: Seq[(String, String)]): Future[Seq[GeographicTaxonomyExt]] = {
     Future.traverse(geoTaxonomycodes)(getExtByCode)
+  }
```

Comment:
The parenthesis added in this file are useless

Reply:
removed in the following commit :)